code/game/sound.dm

```diff
@@ -1,3 +1,8 @@
+
+///Default override for echo
+/sound
+	echo = list(0, 0, -10000, 0, 0, 0, 0, 0.25, 1.5, 1, 0, 1, 0, 0, 0, 0, 1, 0)
```

Comment:
Is this the values that cancel?

if so, can you please document them with a better comment?

Reply:
yes i will sorry

Reply:
```dm
sound.echo = list(
	0, // Direct
	0, // DirectHF
	-10000, // Room
	-10000, // RoomHF
	0, // Obstruction
	0, // ObstructionLFRatio
	0, // Occlusion
	0.25, // OcclusionLFRatio
	1.5, // OcclusionRoomRatio
	1.0, // OcclusionDirectRatio
	0, // Exclusion
	1.0, // ExclusionLFRatio
	0, // OutsideVolumeHF
	0, // DopplerFactor
	0, // RolloffFactor
	0, // RoomRolloffFactor
	1.0, // AirAbsorptionFactor
	0, // Flags (1 = Auto Direct, 2 = Auto Room, 4 = Auto RoomHF)
)
```